src/sys/resource.rs

```diff
@@ -0,0 +1,48 @@
+use std::mem;
+
+use libc::{self, c_int};
+
+use {Errno, Result};
+
+#[repr(i32)]
+pub enum Resource {
```

Comment:
I would prefer `Resource`, because it very clearly expresses what it stands for.